Vagrantfile

```diff
@@ -0,0 +1,168 @@
+        sudo systemctl reboot
+      SHELL
+    # End fedora-atomic
+    end
+
+  # Development commissaire server and services
+  # NOTE: This must start after etcd.
+  config.vm.define "commissaire", primary: true do |commissaire|
+    commissaire.vm.box = "fedora/24-cloud-base"
+    commissaire.vm.network "private_network", ip: "192.168.152.100"
+    config.vm.synced_folder ".", "/vagrant", disabled: true
+    config.vm.synced_folder ".", "/vagrant/commissaire",  mount_options: ['vers=3']
+    config.vm.synced_folder "../commissaire-http", "/vagrant/commissaire-http",  mount_options: ['vers=3']
+    config.vm.synced_folder "../commissaire-service", "/vagrant/commissaire-service",  mount_options: ['vers=3']
+    commissaire.vm.provision "shell", inline: <<-SHELL
+      echo "==> Setting hostname"
+      sudo hostnamectl set-hostname commissaire
+      echo "===> Updating the system"
+      #sudo dnf update -y
+      echo "===> Installing OS dependencies"
+      sudo dnf install -y --setopt=tsflags=nodocs rsync openssh-clients redhat-rpm-config python3-virtualenv gcc libffi-devel openssl-devel git nfs-utils
+      echo "===> Setting up virtualenv"
+      virtualenv-3 commissaire_env
+      echo "===> Installing commissaire"
+      . commissaire_env/bin/activate && pip install -U -r /vagrant/commissaire/test-requirements.txt
+      . commissaire_env/bin/activate && pip install -e /vagrant/commissaire/
+      echo "===> Installing commissaire-http"
+      . commissaire_env/bin/activate && pip install -U -r /vagrant/commissaire-http/test-requirements.txt
+      . commissaire_env/bin/activate && pip install -e /vagrant/commissaire-http/
+      echo "===> Installing commissaire-service"
+      . commissaire_env/bin/activate && pip install -U -r /vagrant/commissaire-service/test-requirements.txt
+      . commissaire_env/bin/activate && pip install -e /vagrant/commissaire-service/
+
+      echo "===> Setting up commissaire-server to autostart"
+      sudo cp /vagrant/commissaire-http/conf/systemd/commissaire-server.service /etc/systemd/system/commissaire-server.service
+      sudo chmod 644 /etc/systemd/system/commissaire-server.service
+      sudo mkdir --parents /etc/commissaire
+      sudo cp /vagrant/commissaire-http/conf/commissaire.conf /etc/commissaire/commissaire.conf
+      sudo sed -i 's|"listen-interface": "127.0.0.1"|"listen-interface": "192.168.152.100"|g' /etc/commissaire/commissaire.conf
+      sudo sed -i 's|"bus-uri": "redis://127.0.0.1:6379/"|"bus-uri": "redis://192.168.152.101:6379/"|g' /etc/commissaire/commissaire.conf
+      sudo sed -i 's|^ExecStart=.*|ExecStart=/bin/bash -c ". /home/vagrant/commissaire_env/bin/activate \\&\\& commissaire-server -c /etc/commissaire/commissaire.conf"|' /etc/systemd/system/commissaire-server.service
+      sudo sed -i 's|Type=simple|\&\\nWorkingDirectory=/vagrant|' /etc/systemd/system/commissaire-server.service
+      echo "===> Starting commissaire-server"
+      sudo systemctl daemon-reload
```

Comment:
Could the `daemon-reload` be done just once at the very end after all the config tweaks are done, or do the services have to come up in sequence?  I guess I've never tested that.  Might shave a second or two off if we can get away with it.